backend/src/main/resources/schema.sql

```diff
@@ -0,0 +1,50 @@
+set foreign_key_checks = 0;
+
+drop table if exists `user` cascade;
+create table `user`
+(
+    user_id       bigint(10) not null auto_increment primary key,
+    user_name     char(200)  not null,
```

Comment:
(가벼운 의견이니 그냥 넘어가도 좋습니다.)
아래처럼 comment로 DB에도 어떤 필드인지 명시해주는 것이 좋은 습관이고, 도메인에 대해 빠르게 파악하기 좋다고 생각합니다.

```suggestion
    user_name     char(200)  not null COMMENT '사용자 이름',
```

Reply:
SQL 문 내에 comment 를 달 수 있는 것은 처음 알게되었습니다.. 감사합니다!